src/app/lib/config/services/mundo.js

```diff
@@ -16,7 +16,7 @@ import withContext from '../../../contexts/utils/withContext';
 export const service = {
   default: {
```

Comment:
I'm curious, why does Mundo, Brasil, and Türkçe not need the `/articles/` prefix in 
`manifestPath` and `swPath` like Cymru Fyw and Naidheachdan?

Reply:
Because the whole of those services will be on Simorgh, whereas for domestic services they will have other pages in Web Core and only articles in Simorgh